### PR TITLE
Rename variable to parameter or outputValue where applicable

### LINKED ISSUE #222: rename parameters and output
As a developer, I'd like to have a consistent naming in regards to parameters and output values.

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should remove all instances of variables referring to the parameters
- [ ] Should remove all instances of variables referring to the output value
- [ ] Should refer to parameters as such
- [ ] Should refer to output values as such

### PR DESCRIPTION
**Pull request by SanJSp**
Fixes #222 
## Changes
This PR changes all occurrences of variable in the code with either parameter if it's an input or outputValue if it's an output. I will also update all ssots (ouputVariable to outputValue fields) in the database when this PR is approved 💁‍♂️ 

If you want an easy teview, check the commits one by one as I have structured them.

## Checklist before merge

**Developer's responsibilities**
* [x] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [x] **Pull request build** has passed
* [x] **tested locally** (in at least chrome & firefox if frontend)
* [x] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

File: client/src/components/content/ModelerSidebar/ModelerSidebar.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
   applicationChangedHandler,
   taskChangedHandler,
   inputParameterChangeHandler,
-  outputVarNameChangeHandler,
+  outputValNameChangeHandler,
```
Comment (lukashueller) [inline client/src/components/content/ModelerSidebar/ModelerSidebar.jsx:13]: I see that the abbreviation has been used before, but I would still argue for using the long form of value :) 
```suggestion
  outputValueNameChangeHandler,
```

File: client/src/components/content/ModelerSidebar/ModelerSidebar.jsx [hunk 2]
```diff
@@ -106,16 +106,16 @@ const ModelerSidebar = ({ modeler, robotId }) => {
...
               inputParameterChangeHandler(
                 elementState.currentElement.id,
                 newValue
               );
             }}
-            outputVariableName={outputVariableName}
+            outputValueName={outputValueName}
             outputNameUpdated={(newValue) => {
-              outputVarNameChangeHandler(
+              outputValNameChangeHandler(
```
Comment (lukashueller) [inline client/src/components/content/ModelerSidebar/ModelerSidebar.jsx:118]: same with the short form here
```suggestion
              outputValueNameChangeHandler(
```

File: client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.js [hunk 3]
```diff
@@ -220,12 +220,12 @@ const inputParameterChangeHandler = (activityId, value) => {
...
- * @description Gets called when the name of the output variable has been changed and updates
- * the output variables name in the ssot
+ * @description Gets called when the name of the output value has been changed and updates
+ * the output values name in the ssot
  * @param {String} activityId id of the activity selected
- * @param {Object} newValue new value of the output variables name
+ * @param {Object} newValue new value of the output values name
  */
-const outputVarNameChangeHandler = (activityId, newValue) => {
+const outputValNameChangeHandler = (activityId, newValue) => {
```
Comment (lukashueller) [inline client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.js:228]:
```suggestion
const outputValueNameChangeHandler = (activityId, newValue) => {
```

File: client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.js [hunk 4]
```diff
@@ -234,7 +234,7 @@ export {
   applicationChangedHandler,
   taskChangedHandler,
   inputParameterChangeHandler,
-  outputVarNameChangeHandler,
+  outputValNameChangeHandler,
```
Comment (lukashueller) [inline client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.js:237]:
```suggestion
  outputValueNameChangeHandler,
```

File: client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.test.js [hunk 5]
```diff
@@ -62,13 +62,13 @@ describe('Sidebar Functionality: Small Utilities', () => {
...
 
-  it('handle output variable name change', async () => {
+  it('handle output value name change', async () => {
     setOutputValueName.mockImplementation((activityId, newValue) => {
       expect(activityId).toEqual(constants.MOCK_ACTIVITY_ID);
       expect(newValue).toEqual(constants.MOCK_NEW_VALUE);
     });
 
-    outputVarNameChangeHandler(
+    outputValNameChangeHandler(
```
Comment (lukashueller) [inline client/src/utils/modelerSidebarFunctionality/modelerSidebarFunctionality.test.js:71]:
```suggestion
    outputValueNameChangeHandler(
```

**Review by lukashueller — CHANGES_REQUESTED** [review]
Hi Sandro,
This looks like very focused, in-depth work to me. I've gone through all the files so far and I don't see any more wrong identifiers. I just hope now that everything continues to fit! ;) 

I would like you to write out all short forms of Value (`Val`) (see comments => you can assume directly, there are no more occurrences)

I would also like you to fix all typos of Parameter (someone always writes (`paramter`) there). 

Also the variable name `val` still exists in the RobotOverview. Can you please find a suitable identifier for it?

Thank you!

**Review by WolfgangDaniel — APPROVED** [review]
Thanks for a thorough job! Please implement Lukas requested changes before merging

**Review by lukashueller — APPROVED** [review]
looks good! Thanks for addressing my points!

**Pull request merged**